### PR TITLE
docs(customization): added support configuration guide

### PR DESCRIPTION
**Pull request by ciiay**
## Description

Update CUSTOMIZATION.md document to explain how to configure support button.

## Which issue(s) does this PR fix

- Fixes [RHIDP-1141](https://issues.redhat.com/browse/RHIDP-1141): [janus-idp/backstage-plugins] Update or remove support button on dynamic-plugins-info

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [x] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: showcase-docs/customization.md [hunk 1]
```diff
@@ -106,3 +106,31 @@ app:
...
+      - title: Issues
+        icon: github
+        links:
+          - url: https://github.com/backstage/backstage/issues
+            title: GitHub Issues
+      - title: Red Hat Developer Hub
+        icon: chat
+        links:
+          - url: https://discord.gg/backstage-687207715902193673
+            title: '#forum-rhdh'
```
Comment (invincibleJai) [inline showcase-docs/customization.md:130]:
```suggestion
      - title: Issues
        icon: github
        links:
          - url: https://github.com/janus-idp/backstage-showcase/issues
            title: GitHub Issues
```

For example let's point to this repo itself.

Comment (ciiay) [reply showcase-docs/customization.md]: Good catch @invincibleJai . I missed this configure when I set for testing and thanks for the suggestion. I wanted to show two items in the example and added the Janus community link. Let me know how do you think, thanks.
example-support-button

**Review by gashcrumb — APPROVED** [review]

**Review by schultzp2020 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**